src/Form/FastlySettingsForm.php

```diff
@@ -244,6 +256,17 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#open' => TRUE,
     ];
 
+    $key_length = (int) $config->get('cache_tag_hash_length') ?: CacheTagsHashInterface::CACHE_TAG_HASH_LENGTH;
+    $form['purge']['purge_options']['cache_tag_hash_length'] = [
+      '#type' => 'number',
+      '#min' => 4,
+      '#max' => 5,
+      '#title' => $this->t('Cache tag hash length'),
+      '#description' => $this->t('For larger sites, it may be necessary to increase the length of the hashed cache tags (eg. <code>d0f</code>) that are used for the <code>Surrogate-Key</code> header and when purging content. This is due to <a href=":hash_collisions">hash collisions</a> which will result in excessive purging of content if the key length is too short. The current key length of <strong>%key_length</strong> can provide %hash_total unique cache keys. Note that this number should not be as large as the total number of cache tags in your site, just high enough to avoid most collisions during purging. Also you can override this with environment variable FASTLY_CACHE_TAG_HASH_LENGTH.', [':hash_collisions' => 'https://en.wikipedia.org/wiki/Hash_table#Collision_resolution', '%key_length' => $key_length, '%hash_total' => pow(64, $key_length)]),
```

Comment:
```suggestion
      '#description' => $this->t('For sites with more content, it may be necessary to increase the length of the hashed cache tags that are used for the <code>Surrogate-Key</code> header and when purging content. This is due to <a href=":hash_collisions">hash collisions</a> which will result in excessive purging of content if the key length is too short. The current key length of <strong>%key_length</strong> can provide %hash_total unique cache keys. Note that this number should not be as large as the total number of cache tags in your site, just high enough to avoid most collisions during purging. Also you can override this with environment variable FASTLY_CACHE_TAG_HASH_LENGTH.', [':hash_collisions' => 'https://en.wikipedia.org/wiki/Hash_table#Collision_resolution', '%key_length' => $key_length, '%hash_total' => pow(64, $key_length)]),
```

Reply:
Will also wrap FASTLY_CACHE_TAG_HASH_LENGTH here with code tag here

Reply:
Modified and pushed.